### DISCUSSION TITLE
Hint links that follow some pattern

### DISCUSSION THREAD

**Discussion by ghjardim** [qutebrowser/qutebrowser#6588] [Q&A] [answered]
Hello! Is it possible to hint links that follow some pattern (for example, all links that have 'youtube.com' in it)? If yes, can I use multiple filters at the same time (for example, all links that have 'youtube.com' or 'instagram.com')?

**Comment by The-Compiler** [comment, marked as answer]
The [`hints.selector` setting](https://github.com/qutebrowser/qutebrowser/blob/master/doc/help/settings.asciidoc#hints.selectors) lets you add any CSS selector (or rather a list of them) - using [attribute selectors](https://developer.mozilla.org/en-US/docs/Web/CSS/Attribute_selectors) you should be able to match on the links.
